integration/ctr/oci_hooks_tests.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2022 Apple Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
+CONTAINER_NAME="${CONTAINER_NAME:-kataocihookstest}"
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.kata.v2}"
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
```

Comment:
hmmm... I don't know the (bad, if any) consequences of setting those options on bats tests.